### PR TITLE
#120: API core view LayoutStride transition

### PR DESCRIPTION
**Pull request by marcinwrobel1986**
API/core/view/layoutStride transition from .md to .rst

before (.md) | after (.rst)
:----------------: | :-------------:
image | image
image | image
image | image

### CODE REVIEW

**Review by fnrizzi — APPROVED** [review]

File: docs/source/API/core/view/layoutStride.rst [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+      An array containing the stride for each dimension of the Layout
+
+   .. rubric:: Typedefs
+
+   .. cppkokkos:type:: array_layout
+
+      A tag signifying that this models the Layout concept
+
+   .. rubric:: Enums
```
Comment (Rombur) [inline docs/source/API/core/view/layoutStride.rst:41]: I would remove this line. It's `is_extent_constructible` is not an enum

File: docs/source/API/core/view/layoutStride.rst [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+   .. cppkokkos:member:: static constexpr bool is_extent_constructible = false;
```
Comment (Rombur) [inline docs/source/API/core/view/layoutStride.rst:43]: Move this line to the `Public Class Members` rubric

File: docs/source/API/core/view/layoutStride.rst [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+      A boolean enum to allow detection that this class is extent constructible
```
Comment (Rombur) [inline docs/source/API/core/view/layoutStride.rst:45]:
```suggestion
      A boolean to allow detection that this class is extent constructible
```

File: docs/source/API/core/view/layoutStride.rst [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+    Kokkos::View<float***> full_mesh; // an entire mesh
+    Kokkos::View<float**, Kokkos::LayoutStride> mesh_subcomponent;
+    mesh_subcomponent = Kokkos::subview(full_mesh,Kokkos::ALL(), 0, Kokkos::ALL()); // take x and z components
+
+Description
+-----------
+
+.. cpp:class:: LayoutStride
+
```
Comment (ldh4) [inline docs/source/API/core/view/layoutStride.rst:24]:
```suggestion
Usage
-----

.. code-block:: c++

    Kokkos::View<float***> full_mesh; // an entire mesh
    Kokkos::View<float**, Kokkos::LayoutStride> mesh_subcomponent;
    mesh_subcomponent = Kokkos::subview(full_mesh,Kokkos::ALL(), 0, Kokkos::ALL()); // take x and z components

Description
-----------

.. cpp:class:: LayoutStride

   When provided to a multidimensional View, lays out memory with an arbitrary stride.
   Most frequently encountered when taking a noncontiguous subview of some larger view.

```

To stay consistent with #203 and #204 in terms of formatting.

**Pull request merged**